### DISCUSSION TITLE
Jamulus 3.8.2 final is now out!

### DISCUSSION THREAD

**Discussion by gilgongo** [jamulussoftware/jamulus#2410] [Announcements]
### This is mainly a bug fix release with the following enhancements:

- Added selection option for level meter style 
- More robust installation behaviour for Windows
- Added an option always to show your own fader first
- Better display of URLs in the chat window

**Please find all the details in the [Changelog](https://github.com/jamulussoftware/jamulus/releases/tag/r3_8_2).**

### ATTENTION: If your upgrade fails and you compile, please see the following issue
https://github.com/jamulussoftware/jamulus/discussions/2416#discussioncomment-2216726

## Downloads

_Windows users: Please note that in the first days after release SmartScreen will probably display warnings about this release being unknown upon download and/or execution of the installer. Let us know when you do not see this warning anymore and we will update this announcement accordingly._

**[⭳ Windows](https://github.com/jamulussoftware/jamulus/releases/download/r3_8_2/jamulus_3.8.2_win.exe)** (ASIO version), alternative: [⭳ JACK version](https://github.com/jamulussoftware/jamulus/releases/download/r3_8_2/jamulus_3.8.2_win_jack.exe)
**[⭳ macOS](https://github.com/jamulussoftware/jamulus/releases/download/r3_8_2/jamulus_3.8.2_mac.dmg)** for High Sierra (10.13) to Big Sur (11) 
**[⭳ macOS legacy build](https://github.com/jamulussoftware/jamulus/releases/download/r3_8_2/jamulus_3.8.2_mac_legacy.dmg)** (unsigned) for macOS Sierra (10.12), El Capitan (10.11) or Yosemite (10.10).
**[⭳ Debian/Ubuntu (amd64)](https://github.com/jamulussoftware/jamulus/releases/download/r3_8_2/jamulus_3.8.2-1_ubuntu_amd64.deb
)**, alternative: [⭳ headless version](https://github.com/jamulussoftware/jamulus/releases/download/r3_8_2/jamulus_headless_3.8.2-1_ubuntu_amd64.deb)
**[⭳ Android](https://github.com/jamulussoftware/jamulus/releases/download/r3_8_2/jamulus_3.8.2_android.apk)** (experimental)

[Alternative Sourceforge mirror](https://sourceforge.net/projects/llcon/files/latest/download)
[Source code](https://github.com/jamulussoftware/jamulus/archive/refs/tags/r3_8_2.zip)
***

**Thanks to everyone who did their part to make this release happen:**

Code contributors: @ann0see @atsampson @BLumia @corrados @DavidSavinkoff @djfun @gilgongo @henkdegroot @hoffie @j-santander @mirabilos @ngocdh @npostavs @pljones @Rob-NY @RobyDati @softins

Application translators: @ann0see @BLumia @dzpex @henkdegroot @hoffie @ignotus666 @jose1711 @jujudusud @melcon @pljones @rolamos @SeeLook @softins

Website contributors/translators: @ann0see @gilgongo @henkdegroot @hoffie @ignotus666 @jujudusud

Main Developers: @ann0see @gilgongo @hoffie @pljones @softins

... and lots of people who brought new ideas or suggestions, guided their local colleagues or helped in various other ways!

---

This Discussion thread will be locked in order to keep things organised.

Feedback, questions or suspected bug reports are appreciated nevertheless -- please start a [new Discussion](https://github.com/jamulussoftware/jamulus/discussions/new) on Github for them.

**Comment by hoffie** [comment]
Heads up to **Debian/Ubuntu** users:
We've just replaced the original 3.8.2 .deb files with the new versions (3.8.2-1) to fix a packaging problem. If you've had issues installing 3.8.2 on Ubuntu 18.04 (`libgcc-s1`), please re-download. There are no other changes. The UI will continue to say 3.8.2 without -1. No 3.8.2-1 releases are planned for any other platforms.

See #2422 for details.
